### DISCUSSION TITLE
Pterodactyl Docker Image

### DISCUSSION THREAD

**Discussion by Maelep** [wolveix/satisfactory-server#31] [Q&A] [answered]
Have you thought about developing an egg/Docker Image for Pterodactyl?

I developed an egg/Docker Image for Rust but I don't think I have enough knowledge to adapt your code to make it compatible with Pterodactyl.

Here is the doc: https://pterodactyl.io/community/config/eggs/creating_a_custom_image.html

**Comment by wolveix** [comment, marked as answer]
Yes, this was discussed in one of the older issues. I'm certainly open to doing so; however, I just haven't had the free time to work on it recently unfortunately. As a Docker image already exists, deploying this server on a non-Pterodactyl server is simple. So while I'd like to support it, the project is easy enough to deploy right now, so it's not a top priority.

Pull requests are always welcome though!

**Reply by Maelep** [reply]
Thank you for your answer.
I would like to work on a Pterodactyl Egg but I can't because I only own Epic Games version and I can't afford the Steam version.
I'll work on this if I ever get the chance to afford the Steam version.
Thank you for your work :)

**Reply by wolveix** [reply]
@Maelep as you no longer need to own the Steam version, it'd be great if you were still willing to work on this!

**Reply by Maelep** [reply]
Great news! I'll check that! Do you have Discord? I would be glad to discuss with you if I need tome information about your project. My Discord is Maelep#7037
